### PR TITLE
Test disputability under different confidence thresholds

### LINKED ISSUE #61: DVM May Not Pick Up all Disputable Values
Tested some bad values today, and the DVM did not believe that they were disputable. No texts were sent or received.

Example Test submission $666 for eth/usd here: `https://goerli.etherscan.io/tx/0xb40f1498d487a9a5ff18cc87f332aa112151190679fe014a107db9b8d42065f1`
(the price of eth/usd today is around $1,200)

dashboard / log output: 
`|  6 | 1668616008 | https://goerli.etherscan.io/tx/0xb4... | ETH | USD |   666    | no ✔️        |`

Version `commit# daa0a2f6b033700cf2182b78ff46a4a0f19335fe`

### PR DESCRIPTION
**Pull request by tallywiesenberg**
Closes #61 

I have found that the DVM is operating as per expected. It has a very wide default confidence threshold, though, by default.

### CODE REVIEW

**Comment by spuddyminer** [comment]
On it! 😀

File: tests/test_data.py [hunk 1]
```diff
@@ -168,3 +168,24 @@ def test_get_contract_iofo():
...
     assert not abi
+
+
+@pytest.mark.asyncio
+async def test_different_conf_thresholds():
+    """test if a value is dispuable under different confindence thresholds"""
+
+    # ETH/USD
+    query_id = "83a7f3d48786ac2667503a61e8c415438ed2922eb86a2906e4ee66d9a2ce4992"
+    val = 666
```
Comment (oraclown) [inline tests/test_data.py:179]: 😮

File: tests/test_data.py [hunk 2]
```diff
@@ -168,3 +168,24 @@ def test_get_contract_iofo():
...
+    # Is disputable
+    disputable = await is_disputable(val, query_id, threshold)
+    assert isinstance(disputable, bool)
+    assert disputable
+
+    threshold = 0.50
+    # Is now not disputable
+    disputable = await is_disputable(val, query_id, threshold)
+    assert isinstance(disputable, bool)
+    assert not disputable
```
Comment (oraclown) [inline tests/test_data.py:191]: Does this pass? 666 is like 55% difference from current price of eth (1172).

**Review by oraclown — APPROVED** [review]
Nice additions, @tallywiesenberg ! All good if tests r passing

File: tests/test_data.py (hunk 2 above)
Comment (tallywiesenberg) [reply tests/test_data.py:202]: All tests pass :)

**Pull request merged**